tasks-by-users/Naufil-Shemle/02-html-code-with-harry/html-tut12.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Semantic Tags</title>
+  </head>
+
+  <body>
+    <h3>Semantic Elements</h3>
+    <details>
+      <summary>
+        ipsum dolor, sit amet consectetur adipisicing elit. Nulla necessitatibus
+        voluptates iste doloremque illo
+      </summary>
```

Comment:
what does this `summary` tag do ?

Reply:
it will show some content which want to  show before clicking on that so we will use summary tag